examples/calendar/backend/models/appointment.model.ts

```diff
@@ -0,0 +1,10 @@
+import { Model } from "sequelize";
+
+export class Appointment extends Model {
+  public id!: number;
```

Comment:
please make model and db inits consistent, ? instead of ! on optional fields, i think ! says its required?

Reply:
Updated